pkg/common/volume_operations_test.go

```diff
@@ -221,42 +243,36 @@ func Test_handleVolumeInProgress(t *testing.T) {
 			Usage:             apiV1.VolumeUsageInUse,
 		}
 		createdVolume *api.Volume
-		err           error
 	)
-
-	// expect volume with "creating" CSIStatus, AC with HDDLVG exists and LogicalVolumeGroup has "created" status
-	svc = setupVOOperationsTest(t)
-	svc.acProvider = acProvider
-	recreatedAC := acToReturn
-	recreatedAC.Spec.StorageClass = requiredSC
-	capMBuilder, capMMock := getCapacityManagerMock()
-	svc.capacityManagerBuilder = capMBuilder
-	capMMock.On("PlanVolumesPlacing", ctxWithID, mock.Anything).
-		Return(buildVolumePlacingPlan(testNode1Name, &expectedVolume, &acToReturn), nil).Times(1)
-	acProvider.On("RecreateACToLVGSC", ctxWithID, requiredSC, &acToReturn).
-		Return(&recreatedAC).Times(1)
-
-	ctx := context.WithValue(testCtx, util.VolumeInfoKey, testNS)
-	createdVolume, err = svc.CreateVolume(ctx, api.Volume{
+	testPVC.ObjectMeta.Name = volumeID
+	assert.Nil(t, svc.k8sClient.Create(ctxWithID, testPVC))
+	assert.Nil(t, svc.k8sClient.CreateCR(ctxWithID, acToReturn.Name, acToReturn))
+	assert.Nil(t, svc.k8sClient.CreateCR(ctxWithID, acrToReturn.Name, acrToReturn))
+	tv := api.Volume{
 		Id:           volumeID,
 		StorageClass: requiredSC,
-		NodeId:       requiredNode,
 		Size:         requiredBytes,
-	})
+		Location:     testLVG.Spec.Name,
+	}
+
+	ctx := context.WithValue(testCtx, util.VolumeInfoKey, &util.VolumeInfo{Name: volumeID, Namespace: testNS})
+	createdVolume, err := svc.CreateVolume(ctx, tv)
 	assert.Nil(t, err)
 	assert.NotNil(t, createdVolume)
-	assert.Equal(t, expectedVolume, *createdVolume)
+	assert.Equal(t, expectedVolume, createdVolume)
```

Comment:
Is `assert.Equal` compare pointers by value by default?

Reply:
I dont get.

Reply:
Got it, thank you!